### PR TITLE
migrate a devon4j project to the latest version

### PR DESCRIPTION
**Pull request by suvmanda**
migrate a devon4j project to the latest version

# What will the user learn in your tutorial?

# What should the user already know when starting the tutorial?

### CODE REVIEW

File: devon4j-migrate/index.asciidoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The java commandlet allows to setup Java. Also it supports devon4j. The arguments (devon java «args») are explained by the below details: 
+https://devonfw.com/website/pages/docs/devonfw-ide-usage.asciidoc.html#java.asciidoc
+
+This tutorial mainly focuses on the migrate a devon4j project to the latest version.
+## Prerequisites
+* devonfw IDE
+
+## Learning goals.
+Here in this tutorial you will learn 
+* How to migrate an older version of devon4j project to the latest version.
```
Comment (sujith-mn) [inline devon4j-migrate/index.asciidoc:12]: Please mention here: Install older version of devonfw ( "2020.08.001")

File: devon4j-migrate/index.asciidoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+installDevonfwIde(["java","mvn"], "2020.08.001")
+--
+====
+Now, we will create sample devon4j application with name *sampleapp*. 
+
+[step]
+--
+createDevon4jProject("sampleapp")
+--
+
```
Comment (sujith-mn) [inline devon4j-migrate/index.asciidoc:27]: apply the command: cd sampleapp

File: devon4j-migrate/index.asciidoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Example for migrating a devon4j application. In command promt just type "devon java migrate".
```
Comment (sujith-mn) [inline devon4j-migrate/index.asciidoc:28]: "Example for migrating a devon4j application." not required..
check the spelling prompt

File: devon4j-migrate/index.asciidoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+This will migrate current devon4j application to the latest version available.
+
```
Comment (sujith-mn) [inline devon4j-migrate/index.asciidoc:30]: This will migrate current devon4j application from  "2020.08.001" to the latest version available.

**Review by sujith-mn — CHANGES_REQUESTED** [review]

File: devon4j-migrate/index.asciidoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+--
+====
+Now, we will create a sample devon4j application with the name *sampleapp*.
+
+[step]
+--
+createDevon4jProject("sampleapp")
+--
+
+In command prompt just type "devon java migrate".
```
Comment (SchettlerKoehler) [inline devon4j-migrate/index.asciidoc:28]: Please contact @EduardKrieger on https://github.com/devonfw-tutorials/tutorial-compiler/pull/154 so you can use it here

File: devon4j-migrate/index.asciidoc (hunk 2 above)
Comment (denise-khuu) [reply devon4j-migrate/index.asciidoc:33]: add the {"dir": "sampleapp"} as parameter

File: devon4j-migrate/index.asciidoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+[step]
+--
+restoreDevonfwIde(["java","mvn"], "2020.08.001")
+--
+
+Now, we will create a sample devon4j application with the name *sampleapp*.
+
+[step]
+--
+createDevon4jProject("sampleapp")
```
Comment (denise-khuu) [inline devon4j-migrate/index.asciidoc:25]: use 'com.example.sampleapp'

Comment (denise-khuu) [reply devon4j-migrate/index.asciidoc:25]: or com.example.application.sampleapp

**Comment by SchettlerKoehler** [comment]
What is the state of this PR?

**Pull request closed without merging**